src/praisonai-agents/tests/unit/test_gateway_degraded_state.py

```
@@ -0,0 +1,95 @@
+)
+
+
+def test_owner_is_frozen_and_serialises():
+    owner = DegradedOwner(
+        owner_kind="provider",
+        owner_id="openai",
+        state="cold",
+        reason="auth rejected (401)",
+        retry_hint="re-set OPENAI_API_KEY",
+    )
+    with pytest.raises(Exception):
```

Ruff (0.16.0)

[warning] Do not assert blind exception: `Exception`

(B017)